state/cleanup.go

```diff
@@ -21,8 +21,8 @@ import (
 	"github.com/juju/juju/core/objectstore"
 	coreunit "github.com/juju/juju/core/unit"
 	applicationerrors "github.com/juju/juju/domain/application/errors"
+	internalrelation "github.com/juju/juju/domain/relation"
```

Comment:
We don't want this alias now.

Reply:
That slipped through, checked for others and that was the only one